### PR TITLE
[Gui] GUI documentations

### LINKED ISSUE #22: [Doc] GUI Documentation
We need documentation for our GUI module.
Follow sphinx-napoleon plugin.

For reference, [Check this out](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

### PR DESCRIPTION
**Pull request by chet0322**
- [ ] closes #22
- [ ] tests added / passed
- [ ] documentation done

### CODE REVIEW

**Comment by chet0322** [comment]
Will finish documentation by tonight

File: captslog/ui/mainwindow.py [hunk 1]
```diff
@@ -25,34 +17,39 @@ def _translate(context, text, disambig):
...
-
-        # Size Policy
-        sizePolicy = QtGui.QSizePolicy(
-            QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
-        sizePolicy.setHorizontalStretch(0)
-        sizePolicy.setVerticalStretch(0)
-        sizePolicy.setHeightForWidth(
-            self.sizePolicy().hasHeightForWidth())
+		self.setObjectName(_fromUtf8("MainWindow"))
+
```
Comment (jaehoonhwang) [inline captslog/ui/mainwindow.py]: No space.

File: captslog/ui/mainwindow.py [hunk 2]
```diff
@@ -25,34 +17,39 @@ def _translate(context, text, disambig):
...
-        # Size Policy
-        sizePolicy = QtGui.QSizePolicy(
-            QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding)
-        sizePolicy.setHorizontalStretch(0)
-        sizePolicy.setVerticalStretch(0)
-        sizePolicy.setHeightForWidth(
-            self.sizePolicy().hasHeightForWidth())
+		self.setObjectName(_fromUtf8("MainWindow"))
+		
+		"""Set up the size of the Window
```
Comment (jaehoonhwang) [inline captslog/ui/mainwindow.py]: Add a period

Comment (jaehoonhwang) [inline captslog/ui/mainwindow.py]: There shouldn't be a documentation here.
Documentations are for functions and classes.

File: captslog/ui/mainwindow.py [hunk 3]
```diff
@@ -25,34 +17,39 @@ def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
 
-# Main Window GUI
 class Ui_MainWindow(QtGui.QMainWindow):
 
     def setupUi(self, MainWindow):
-        """Setting up Main window UI.
```
Comment (jaehoonhwang) [inline captslog/ui/mainwindow.py]: This is where the documentation should be.

File: captslog/ui/centralwidget.py [hunk 4]
```diff
@@ -17,16 +17,25 @@ def _translate(context, text, disambig):
     def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
-#Collection of all widgets
 class CentralWidget(QtGui.QWidget):
+	"""Set up the central GUI
```
Comment (jaehoonhwang) [inline captslog/ui/centralwidget.py]: You don't setup CentralGUIWidget here (You do that at the constructor.
Just describe what class this is or name it.

File: captslog/ui/centralwidget.py [hunk 5]
```diff
@@ -17,16 +17,25 @@ def _translate(context, text, disambig):
...
+	
+	The CentralWidget consist of three main widgets including the
+	journal list, journal entry, and journal view.
+	
+	"""
 	def __init__(self, parent):
```
Comment (jaehoonhwang) [inline captslog/ui/centralwidget.py]: document __init__

File: captslog/ui/centralwidget.py [hunk 6]
```diff
@@ -58,9 +72,9 @@ def __HorizLayout(self):
 			
 		self.setLayout(self.horizontalLayout_7)
 		
-	
-	# Journal List
 	def __JournalList(self):
+		"""Journal List widget
```
Comment (jaehoonhwang) [inline captslog/ui/centralwidget.py]: It should be """Journal List widget."""

File: captslog/ui/centralwidget.py (hunk 4 above)
Comment (jaehoonhwang) [inline captslog/ui/centralwidget.py]: Also you need to put period after the first sentence.
such as 

```py
def blahblah(self):
    """Blahblah. 

     BLASHLHALF


    """
```

File: captslog/ui/centralwidget.py [hunk 7]
```diff
@@ -17,16 +17,25 @@ def _translate(context, text, disambig):
...
+	"""
 	def __init__(self, parent):
 		super(CentralWidget, self).__init__(parent)
 		self.__CentralWidget()
 		self.__HorizLayout()
 		
-	
-	#Central Widget
 	def __CentralWidget(self):
+		"""Set up central widget
```
Comment (jaehoonhwang) [inline captslog/ui/centralwidget.py]: period.

File: captslog/ui/main.py [hunk 8]
```diff
@@ -23,12 +23,22 @@ def _translate(context, text, disambig):
 class Main(QtGui.QMainWindow):
 
     def __init__(self, parent=None):
+		"""
```
Comment (jaehoonhwang) [inline captslog/ui/main.py]: Over indented.

File: captslog/ui/main.py [hunk 9]
```diff
@@ -23,12 +23,22 @@ def _translate(context, text, disambig):
...
+		Args:
+			ui (QMainWindow) : The instance of Ui_MainWindow class.
+		"""
         super(Main, self).__init__(parent)
         self.ui = Ui_MainWindow()
         self.ui.setupUi(self)
         self.ui.center_widget.entry.journalEntry.textChanged.connect(self.text_triggered)
 
     def text_triggered(self):
+		"""Update text to markdown text simultaneously
```
Comment (jaehoonhwang) [inline captslog/ui/main.py]: Over indent

**Comment by jaehoonhwang** [comment]
By the way this PR should wait until @JeewakaSankalpa 's PR is mereged

**Comment by chet0322** [comment]
The new documentation is done.

**Pull request merged**